src/Compiler/AbstractIL/il.fs

```diff
@@ -1451,6 +1463,9 @@ type ILLocalDebugInfo =
         DebugMappings: ILLocalDebugMapping list
     }
 
+    override x.ToString() =
+        (fst x.Range).ToString() + "-" + (snd x.Range).ToString()
```

Comment:
Each range (type) access may involve filesystem access, I would be careful here.